packages/api/src/localization/en-US.json

```diff
@@ -106,9 +105,12 @@
   "TOAST_DISMISS_BUTTON": "Dismiss",
   "TOAST_TITLE_ALT": "Notification",
   "_TOAST_TITLE_ALT.comment": "This is for screen reader. The label for individiual notification.",
+  "TRANSCRIPT_ARIA_LABEL_ALT": "Transcript, press arrow keys to navigate.",
```

Comment:
```suggestion
  "TRANSCRIPT_ARIA_LABEL_ALT": "Transcript; press arrow keys to navigate.",
```

Reply:
'Transcript is focused; press arrow keys to navigate inside the transcript'
'Transcript is focused; press arrow keys to navigate inside'

Reply:
After some thoughts, I think we don't need "is focused". This is because the end-user need to intentionally move the focus to transcript by <kbd>TAB</kbd>.